sync-argocd-applications/README.md

```diff
@@ -0,0 +1,37 @@
+# sync-argocd-applications
```

Comment:
I'd suggest `argocd-sync-applications` naming (to have a class of `argocd*` actions dealing with ArgoCD stuff). same as `yq-yaml-parser` is dealing with `yq`

Reply:
It is a better naming indeed.